addons/ondevice-knobs/package.json

```diff
@@ -26,7 +26,7 @@
     "prop-types": "^15.6.2",
     "react-native-color-picker": "^0.4.0",
     "react-native-modal-datetime-picker": "^5.1.0",
```

Comment:
consider bumping `react-native-modal-datetime-picker` too, this will be required for the 0.59 upgrade. If that's too much outside of the scope of this PR I can take a stab at it later.

Reply:
@benoitdion Tomorrow morning 08:00 UTC +1,

We're going on a chaos programming session to fix up Storybook for RN.
@igor-dv and I will be there, and we've invited a few more.

Goal is to get the server part working well again.
Secondary goal would be to upgrade a lot to the latest version, possibly babel etc.

If you'd be interested in helping, you're welcome to join, give advice, possibly learn abit about what's going on in our codebase?

Join our discord and PM me if you want an invite to the Zoom meeting.

Reply:
I'd LOVE a comprehensive PR upgrading RN to the latest and greatest of all versions